ethcore/light/src/client/mod.rs

```diff
@@ -263,6 +266,30 @@ impl Client {
 		for verified_header in self.queue.drain(MAX) {
 			let (num, hash) = (verified_header.number(), verified_header.hash());
 
+			// Verify Block Family
+			let parent_header = match self.chain.block_header(BlockId::Hash(*verified_header.parent_hash())) {
+				Some(header) => header,
+				None => continue, // skip import of block with missing parent.
+			};
+
+			let verify_family_result = self.engine.verify_block_family(&verified_header, &parent_header.decode(), None);
+			if let Err(e) = verify_family_result {
+				warn!(target: "client", "Stage 3 block verification failed for #{} ({})\nError: {:?}",
+					verified_header.number(), verified_header.hash(), e);
+				bad.push(hash);
+				continue;
+			};
+
+			// "external" verification.
+			let verify_external_result = self.engine.verify_block_external(&verified_header, None);
+			if let Err(e) = verify_external_result {
+				warn!(target: "client", "Stage 4 block verification failed for #{} ({})\nError: {:?}",
```

Comment:
I think this is more readable as an exception, where the format string is on one line and the arguments are below it, in order. 
It makes for easy visual matching of format args to values without much vertical scanning.

The difference between it and a regular function call with many params is that you tend to look back and forth much more with a log.